about.html

```diff
@@ -0,0 +1,161 @@
+    <!-- JavaScript Bundle with Popper -->
+    <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+    <script defer src="js/menu.js"></script>
+    <script defer src="js/about.js"></script>
+</head>
+
+<body>
+    <header class="nav-container">
+        <div class="container-fluid header1">
+            <ul class="row">
+                <li class="col-md-6"></li>
+                <li class="col-md-1">
+                    <a href="#"><i class="fa fa-facebook"></i></a>
+                </li>
+                <li class="col-md-1">
+                    <a href="# "><i class="fa fa-twitter"></i></a>
+                </li>
+                <li class="col-md-1">English</li>
+                <li class="col-md-1">My Page</li>
+                <li class="col-md-1">Logoout</li>
+                <li class="col-md-1"></li>
+            </ul>
+        </div>
+        <div class="container-fluid header2">
+            <i class="mt-5 hamburger-menu openMenu fa fa-bars"></i>
+            <ul class="row header2List mainMenu">
+                <li class="closeMenu"><i class="closeMenu fa fa-times"></i></li>
+                <li class="d-none d-md-block col-md-3 p-3">
+                    <a href="./index.html#AWECampaine" class="logo"><img src="./image/logo.png" alt="women image logo" />
+                    </a>
+                </li>
+                <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+                    <a class="nav-link" href="./index.html#about">About<span class="navSpanArrow1">></span>
+            </a>
+                </li>
+                <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+                    <a class="nav-link" href="./index.html#programs">Program<span class="navSpanArrow2">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+            <a class="nav-link" href="./index.html#speakers"
+              >Join<span class="navSpanArrow3">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+            <a class="nav-link" href="./index.html#sponsor"
+              >Sponsor<span class="navSpanArrow2">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+            <a class="nav-link" href="#news"
+              >News<span class="navSpanArrow4">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-2 py-3 EAWDicore nav-item">
+            <a class="nav-link" href="./index.html#AWECampaine">
+              EAW Campaign
+              <span class="navSpanArrow5">></span>
+            </a>
+                </li>
+                <li class="col-md-1"></li>
+            </ul>
+        </div>
+    </header>
+    <section class="bodyHeader container">
```

Comment:
The contents your page for desktop versions needs some alignments. Kindly note that we as much as possible want our design to closely mimic the provide [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), therefore kindly improve upon the following aspect of alignment: of your design:
- [ ] The contents of the hero section should be aligned to the left together with the logo 
- [ ] The right part (the rectangle with red thick border) of the white navbar should be aligned to the right together with logout text on the dark navbar 

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/182956414-af34ec61-842d-4998-bfda-fb65a9759a15.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/182956688-bb1e07e8-9b0f-4702-8b76-e133fad27940.png)